lib/facter/nginx_version.rb

```diff
@@ -2,7 +2,7 @@
   setcode do
     if Facter.value('kernel') != 'windows' && Facter::Util::Resolution.which('nginx')
       nginx_version = Facter::Util::Resolution.exec('nginx -v 2>&1')
-      %r{nginx version: nginx\/([\w\.]+)}.match(nginx_version)[1]
+      %r{nginx version: (nginx|openresty)\/([\w\.]+)}.match(nginx_version)[1]
```

Comment:
Needs to be `%r{nginx version: (nginx|openresty)\/([\w\.]+)}.match(nginx_version)[2]` ?